packages/dds/tree/src/test/snapshots/schema-files/recursiveType-empty.json

```diff
@@ -1,158 +1,6 @@
 {
   "version": 1,
   "nodes": {
-    "com.fluidframework.leaf.boolean": {
```

Comment:
due to how the schema for these tests are now specified (by converging the simple-tree schema), they no longer end up with lots of irrelevant schema from the flex tree schema builder from other test cases.